src/routes/admin/_DictionaryRow.svelte

```diff
@@ -25,24 +27,46 @@
         if (confirm("Flip this dictionary's visibility?")) {
           dispatch('toggleprivacy');
         }
-      }}>
+      }}
+    >
       {dictionary.public ? 'Public' : 'Private'}
     </Button>
   </td>
   <td class="italic">
     <DictionaryFieldEdit field={'name'} value={dictionary.name} dictionaryId={dictionary.id} />
   </td>
+  <td
```

Comment:
I wrapped up my new RolesManagment component inside your Collection component because I think it's very handly. But I have noticed it takes a long time to update the data of the new columns.
I tried to make everything else as reusable and consistent as I can. I almost finish but I need to do some more adjustments